test/typescript-karma/Dashboards/Component/datagrid.js

```diff
@@ -6,9 +6,10 @@ import EditMode from '../../../../code/dashboards/es-modules/masters/modules/lay
 Dashboards.DataGridPlugin.custom.connectDataGrid(DataGrid);
 Dashboards.PluginHandler.addPlugin(Dashboards.DataGridPlugin);
 
-const { test } = QUnit;
+const { skip } = QUnit;
 
-test('DataGrid component with dataTable', async function (assert) {
+// DataGrid1 Tests to update
+skip('DataGrid component with dataTable', async function (assert) {
```

Comment:
What is the reason of skipping test?

Reply:
Tests for the DataGrid Component are also part of this task: https://app.asana.com/0/1203234853779882/1207620146790059/f